aif360/sklearn/datasets/openml_datasets.py

```diff
@@ -247,3 +252,17 @@ def fetch_bank(*, data_home=None, cache=True, binary_age=True, percent10=False,
     return standardize_dataset(df, prot_attr=[age], target='deposit',
                                usecols=usecols, dropcols=dropcols,
                                numeric_only=numeric_only, dropna=dropna)
+
+
+class OpenMLStore(ABC):
+    @abstractmethod
+    def init(self, **kwargs):
+        pass
+
+    def download(self, data_id, data_home):
+        df = fetch_openml(data_id=1590, data_home=data_home or DATA_HOME_DEFAULT,
```

Comment:
Can we change to `data_id=data_id if data_id else 1590` so it's not hardcoded?

Reply:
Sure!

Reply:
@lindawangg Done. Please have a look and let me know if further changes are required.